components/dashboard/src/settings/Integrations.tsx

```diff
@@ -266,19 +266,19 @@ function GitProviders() {
                             &nbsp;
                         </div>
                     </div>
-                    <div className="p-0 my-auto flex flex-col w-3/12">
+                    <div className="p-0 my-auto flex flex-col xl:w-3/12 x-4/12">
                         <span className="my-auto font-medium truncate overflow-ellipsis">{ap.authProviderType}</span>
                         <span className="text-sm my-auto text-gray-400 truncate overflow-ellipsis">{ap.host}</span>
                     </div>
-                    <div className="p-0 my-auto flex flex-col w-2/12">
+                    <div className="p-0 my-auto flex flex-col xl:w-2/12 w-6/12">
                         <span className="my-auto truncate text-gray-500 overflow-ellipsis">{getUsername(ap.authProviderId) || "–"}</span>
                         <span className="text-sm my-auto text-gray-400">Username</span>
                     </div>
-                    <div className="p-0 my-auto flex flex-col w-5/12">
+                    <div className="p-0 my-auto none xl:flex flex-col w-5/12">
```

Comment:
**issue**: This is making alignment a bit worse in this area. I thought we agreed on hiding the permissions column altogether, right? 😭

Reply:
yes. oh, no. this is not intended.

Reply:
it's fixed now. don't know what happened 🤷🏻‍♂️